config/samples/ndb_v1alpha1_database.yaml

```diff
@@ -4,12 +4,12 @@ metadata:
   name: db
 spec:
   ndb:
-    clusterId: "Nutanix Cluster Id"
+    clusterId: [Nutanix Cluster Id]
     credentialSecret : ndb-secret-name
     server: https://[NDB IP]:8443/era/v0.9
     skipCertificateVerification: true
   databaseInstance:
-    databaseInstanceName: "Database Instance Name"
+    databaseInstanceName: [Database Instance Name]
```

Comment:
Enclosing these values in '[ ..... ]' will change the types `clusterId` and `databaseInstanceName` to a list type. I don't think that's required for these fields now, can we revert these changes?

Reply:
The original plan was to keep one single pattern for all the values "that need to be updated". 

as in  "server: https://[NDB IP]:8443/era/v0.9"

But I am ok to revert.